core/pool_vector.h

```diff
@@ -458,7 +458,7 @@ class PoolVector {
 
 	bool is_locked() const { return alloc && alloc->lock > 0; }
 
-	inline const T operator[](int p_index) const;
+	inline T operator[](int p_index) const;
```

Comment:
What was the error reported?

Reply:
A 'const' modifier on a member function return type is useless. It is usually a typo or misunderstanding, since the syntax for a 'const' function is 'int foo() const', not 'const int foo()'.
https://lgtm.com/projects/g/godotengine/godot/alerts/?mode=list&id=cpp%2Fmember-const-no-effect
https://lgtm.com/rules/2156200647/